app/models/chef.rb

```diff
@@ -1,4 +1,14 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+   has_many :dish_ingredients, through: :dishes
+   has_many :ingredients, through: :dishes
```

Comment:
Nice work adding additional associations! Since there is already an association between dishes and ingredients, you can actually consolidate these lines down to `has_many :ingredients, through: :dishes`